charts/kubemq/latest/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+description: Message broker supports message patterns of Active Queue, Command-Query and Pub-Sub events.
+appVersion: "1.0"
+name: kubemq
+keywords:
+- message
+- event
+- queue
+home: https://kubemq.io/
+version: 0.1.0
+sources:
+- https://github.com/kubemq-io/docs
+maintainers:
+- name: ido-sheffer
+  email: ido.sheffer@kubemq.io
+icon: https://account.kubemq.io/assets/img/logo1.png
```

Comment:
would prefer to add the icon to the parent directory and refer it by: `icon: file://../logo.png`, so that air-gapped user can also view it correctly, thanks.

Reply:
hello update a new pull request "Added KubeMQ message broker #256" https://github.com/rancher/charts/pull/256